Make initialize_docker fails

### Describe the problem and steps to reproduce it:
I followed guide that is install docs. When I executing command `Make initialize_docker`, It occurred Error.

### What happened?
```
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/code/src/olympia/landfill/management/commands/generate_default_addons_for_frontend.py", line 77, in handle
    serializer = GenerateAddonsSerializer()
  File "/code/src/olympia/landfill/serializers.py", line 45, in __init__
    self.fxa_email = os.environ['UITEST_FXA_EMAIL']
  File "/usr/local/lib/python3.6/os.py", line 669, in __getitem__
    raise KeyError(key) from None
KeyError: 'UITEST_FXA_EMAIL'
Makefile-docker:97: recipe for target 'populate_data' failed
make: *** [populate_data] Error 1
Makefile-os:57: recipe for target 'initialize_docker' failed
make: *** [initialize_docker] Error 2

```

### What did you expect to happen?
Success make and run server.

### Anything else we should know?
Additionally, I searched keyword **UITEST_FXA_EMAIL**. I found issue that #10888. So I return commit(`git checkout 7f7c8dd`) before this issue. I succeeded `Make initialize_docker` and I can access http://olympia.test.

I tried to run:

```
UITEST_FXA_PASSWORD=uitester UITEST_FXA_EMAIL=uitest-$(uuidgen)@restmail.net make initialize_docker
```

and I hit an error again:

```
[...]

{"Timestamp": 1553689745401614592, "Type": "raven.base.Client", "Logger": "http_app_addons", "Hostname": "cd13583ce15b", "EnvVersion": "2.0", "Severity": 6, "Pid": 2822, "Fields": {"uid": "", "remoteAddressChain": "", "msg": "Raven is not configured (logging is disabled). Please see the documentation for more information."}}
Traceback (most recent call last):
  File "manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 381, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.6/site-packages/django/core/management/__init__.py", line 375, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 323, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.6/site-packages/django/core/management/base.py", line 364, in execute
    output = self.handle(*args, **options)
  File "/code/src/olympia/landfill/management/commands/generate_default_addons_for_frontend.py", line 77, in handle
    serializer = GenerateAddonsSerializer()
  File "/code/src/olympia/landfill/serializers.py", line 45, in __init__
    self.fxa_email = os.environ['UITEST_FXA_EMAIL']
  File "/usr/local/lib/python3.6/os.py", line 669, in __getitem__
    raise KeyError(key) from None
KeyError: 'UITEST_FXA_EMAIL'
Makefile-docker:97: recipe for target 'populate_data' failed
make: *** [populate_data] Error 1
make: *** [initialize_docker] Error 2
```

--- a/Makefile-docker
+++ b/Makefile-docker
@@ -7,8 +7,8 @@ export PIP_COMMAND=pip${PYTHON_VERSION_MAJOR}
 export PYTHON_COMMAND=python${PYTHON_VERSION_MAJOR}
 
 # FXA username and password for ui-tests
-export FXA_EMAIL=${UITEST_FXA_EMAIL}
-export FXA_PASSWORD=${UITEST_FXA_PASSWORD}
+export UITEST_FXA_EMAIL := uitest-$(shell uuid)@restmail.net
+export UITEST_FXA_PASSWORD := uitester
 
 NUM_ADDONS=10
 NUM_THEMES=$(NUM_ADDONS)

